keyboards/alk0/cornedeon/info.json

```
@@ -0,0 +1,97 @@
+{
+    "manufacturer": "ALK0",
+    "url": "https://github.com/alko-kbd/qmk-vial",
+    "maintainer": "qmk",
+    "usb": {
+        "vid": "0x4653"
+    },
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "nkro": true,
+        "oled": false,
+        "rgblight": false,
+        "rgb_matrix": true
+    },
+    "bootmagic": {
+        "matrix": [0, 1]
+    },
+    "split": {
+        "enabled": true,
+        "bootmagic": {
+            "matrix": [4, 1]
+        }
+    },
+    "layout_aliases": {
+        "LAYOUT": "LAYOUT_split_3x6"
+    },
+    "community_layouts": [ "split_3x6" ],
+    "layouts": {
+        "LAYOUT_split_3x6": {
```

```suggestion
    "layouts": {
        "LAYOUT": {
```
There's no matching community layout and you only support one layout, so you might as well just call it `LAYOUT`.